cli/scripts/ace.py

```diff
@@ -183,16 +183,32 @@ def get_key(p_con, p_schema, p_table):
     return ",".join(key_lst)
 
 
+def parse_nodes(nodes) -> list:
+    node_list = []
+    if type(nodes) is str and nodes != "all":
+        node_list = [s.strip() for s in nodes.split(",")]
+    elif type(nodes) is not str:
+        node_list = nodes
+
+    if nodes != "all":
+        rep_check = set(node_list)
+        if len(rep_check) < len(node_list):
+            util.message(
+                f"Some node was given multiple times, ignoring copies",
```

Comment:
Please remove the format string here as well. Also, the warning message can simply say, "Ignoring duplicate node names"